code/modules/atmospherics/machinery/airalarm.dm

```diff
@@ -826,8 +856,11 @@
 	playsound(src, "sparks", 50, 1)
 
 /obj/machinery/airalarm/obj_break(damage_flag)
-	..()
-	update_icon()
+	if(circuit && !(flags_1 & NODECONSTRUCT_1) && !(stat & BROKEN))
+		playsound(src, 'sound/effects/glassbr3.ogg', 100, 1)
```

Comment:
Why'd you remove the ..() call?

Reply:
Oh, that might actually be a me-goof, lemmie just revert it